Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
```

Comment:
The scenario describes something a bit different. What about `Validates that the "Send to trash" button is replaced with "Delete" when browsing users` ?